src/tee/mod.rs

```diff
@@ -1,2 +1,5 @@
 #[cfg(feature = "tee-sev")]
 pub mod sev;
+
+#[cfg(feature = "tee-sev")]
```

Comment:
Should there be a different feature for SNP?

Reply:
Since SEV is backwards compatible with SNP, we've been packing the two together in libkrun. I assumed we would do the same for a KBS server.

@slp thoughts?